<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addtrainDialog</class>
 <widget class="QDialog" name="addtrainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>addtrain</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel {
	font: 9pt &quot;幼圆&quot;;
	font-size:15px;
	color:black;
}

QPushButton {
	font: 9pt &quot;黑体&quot;;
	font-size:17px;
	color:black;
}

QDialog {
	background-color:navy;
}
QLabel {
	color:white;
}</string>
  </property>
  <widget class="QTableWidget" name="stationWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="priceWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>293</width>
     <height>91</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="comfirmButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>620</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>620</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trainIdLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>72</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>train_id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>72</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="catalogLineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>72</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>catalog</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>列车号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>列车名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>列车类别</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>车票种类</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>车站</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>基本信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>161</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增添</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>161</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>161</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>341</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>341</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增添</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>341</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>addtrainDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>829</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
